templates/index.md.tmpl

```diff
@@ -66,6 +66,67 @@ export SPECTROCLOUD_APIKEY=5b7aad.........
 provider "spectrocloud" {}
 ```
 
+## Import
+In Terraform v1.5.0 and later, use an [`import` block](https://developer.hashicorp.com/terraform/language/import)
+
+Each Spectrocloud resource type has its own specific requirements for the import process. It's essential to refer to the documentation for each resource to understand the exact format of the `id` and any other required parameters.
+The `import` block specifies the resource you want to import and its unique identifier with the following structure:
+
+```terraform
+import {
+  to = <resource>.<name>
+  id = "<unique_identifier>"
+}
+```
+
+- `<resource>`: The type of the resource you are importing.
+- `<name>`: A name you assign to the resource within your Terraform configuration.
+- `<unique_identifier>`: The ID of the resource you are importing. This can include additional context if required.
+
+## Examples
+
+1. **Import with Context**: When importing resources that require additional context, the `id` is followed by a context, separated by a colon. For example, importing a cluster with context looks like this:
+
```

Comment:
```suggestion
The following examples showcase how to import a resource. Some resource requires the context to be specified during the import action. The context refers to the Palette scope. Allowed values are either `project` or `tenant`. 

####  Import With Context

When importing resources that require additional context, the `id` is followed by a context, separated by a colon.
```

Instead of bullets, we can use a smaller heading to make it easier to identify the example.